### DISCUSSION TITLE
Download progress bars under the "Other" category when viewing a profile.

### DISCUSSION THREAD

**Discussion by LilithKatsuna** [ebkr/r2modmanPlus#722] [Ideas]
It would be great if downloads we initiate either through Thunderstore directly OR when we click off of a download bar to continue browsing mods that it would appear as a progress bar underneath the "Other" section on the profile screen.

In addition you could also make it so after the download is complete, it will change to "Download Complete!" or something similar, with added options to auto hide once complete in the settings.

This would help users keep track of what they're currently downloading without needing to keep the progress bar open, an amazing quality of life addition that I'm sure a lot of users will be happy about.

**Comment by ebkr** [comment]
Hey, you can view progress after clicking off by clicking the download icon next to the online tab in the navigation bar. Not as flashy but serves the same purpose

**Reply by LilithKatsuna** [reply]
Ah gotchya, well it's still an option if you do decide to take the flashy route lmao
